Content/default/.editorconfig

```diff
@@ -5,4 +5,10 @@ indent_style = space
 indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
-insert_final_newline = false
\ No newline at end of file
+insert_final_newline = false
+
+[*.fs]
+fsharp_space_before_colon=true
+
+[src/Client/Index.fs]
+fsharp_single_argument_web_mode=true
```

Comment:
This is for the Feliz formatting, isn't it? If so we should probably extend it to all fs files under `src/Client`

Reply:
Indeed, this is to have Feliz style. If you have files that do nothing with Feliz this could potentially lead to unexpected results, not in line with the formatting guide. That is why I pinned down the exact file.